deploy/kubernetes/overlays/stable-master/fsgrouppolicy.yaml

```diff
@@ -1,9 +1,3 @@
-$patch: replace
-apiVersion: storage.k8s.io/v1beta1
-kind: CSIDriver
-metadata:
-  name: filestore.csi.storage.gke.io
-spec:
-  attachRequired: false
-  podInfoOnMount: true
-  fsGroupPolicy: File
+- op: add
```

Comment:
Are these changes supposed to be in here too? If so maybe the commit/PR descriptions should be updated.

Reply:
Ah, just saw your comment above. Let's keep the fsgroup policy change in, just update the description.

Reply:
yes I discovered this failure as part of the presubmit run of integration test. so added a new commit in this PR. Will update the description